src/AddTodoForm.jsx

```diff
@@ -1,34 +1,37 @@
-import { useState } from 'react';
-
-
-function AddTodoForm({onAddTodo}) {
-
-    const [todoTitle, setTodoTitle] = useState('');
-
-    const handleTitleChange = (event) => {
-        const newTodoTitle = event.target.value;
-        setTodoTitle(newTodoTitle);
+import React, {useState} from 'react';
+import InputWithLabel from './InputWithLabel';
+
+function AddTodoForm ({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle =event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const newTodo = {
+      title: todoTitle,
+      id: Date.now(),
     };
-
-    const handleAddTodo = (event) => {
-        event.preventDefault();
-
-        console.log(todoTitle);
-        onAddTodo({ title: todoTitle, id: Date.now() });
-        setTodoTitle('');    
-    
-    };
-
+    onAddTodo(newTodo);
+    setTodoTitle("");
+  };
     return (
-        <div>
-            <form onSubmit={handleAddTodo}>
-                <label htmlFor="todoTitle">Title: </label>
-                <input id="todoTitle" type='Text' name='title' value={todoTitle} onChange={handleTitleChange}></input>
-                <button type='submit'>Add</button>
-            </form>
-        </div>
-    );
-    
-}
+        <form onSubmit={handleAddTodo}>
+        <InputWithLabel
+          type="text"
+          id="todoTitle"
+          name="title"
+          value={todoTitle}
+          onChange={handleTitleChange}
+          autoFocus={true}
+          >
+            Title:
+          </InputWithLabel>
+          <button type="submit">Add</button>
+        </form>
+      );
+};
```

Comment:
Remove this unecessary semicolon.
```suggestion
}
```